mscore/exportxml.cpp

```diff
@@ -1156,12 +1156,12 @@ void ExportMusicXml::credits(Xml& xml)
       // determine page formatting
       const PageFormat* pf = _score->pageFormat();
       if (!pf) return;
-      const double h  = getTenthsFromInches(pf->size().height());
-      const double w  = getTenthsFromInches(pf->size().width());
-      const double lm = getTenthsFromInches(pf->oddLeftMargin());
-      const double rm = getTenthsFromInches(pf->oddRightMargin());
-      //const double tm = getTenthsFromInches(pf->oddTopMargin());
-      const double bm = getTenthsFromInches(pf->oddBottomMargin());
+      const double h  = pf->size().height()   * PageFormat::SCALE_XML;
```

Comment:
I did a diff between a MusicXML file before and after this PR. If we exclude rounding it looks correct except for margins, which are inversed and some default-x values which are wrong, probably for the same reason.
```
<         <right-margin>56.6929</right-margin>
---
>         <right-margin>-55.1181</right-margin>
35c35
<         <right-margin>56.6929</right-margin>
---
>         <right-margin>-55.1181</right-margin>
```